vpr/src/base/vpr_api.h

```diff
@@ -52,38 +53,54 @@ bool vpr_flow(t_vpr_setup& vpr_setup, t_arch& arch); //Run the VPR CAD flow
 /*
  * Stage operations
  */
-bool vpr_pack_flow(t_vpr_setup& vpr_setup, const t_arch& arch);    //Perform, load or skip the packing stage
-bool vpr_pack(t_vpr_setup& vpr_setup, const t_arch& arch);         //Perform packing
-void vpr_load_packing(t_vpr_setup& vpr_setup, const t_arch& arch); //Loads a previous packing
 
-bool vpr_place_flow(t_vpr_setup& vpr_setup, const t_arch& arch);     //Perform, load or skip the placement stage
-void vpr_place(t_vpr_setup& vpr_setup, const t_arch& arch);          //Perform placement
-void vpr_load_placement(t_vpr_setup& vpr_setup, const t_arch& arch); //Loads a previous placement
-
-RouteStatus vpr_route_flow(t_vpr_setup& vpr_setup, const t_arch& arch); //Perform, load or skip the routing stage
+/** @brief Perform, load or skip the packing stage */
+bool vpr_pack_flow(t_vpr_setup& vpr_setup, const t_arch& arch);
+/** @brief Perform packing */
+bool vpr_pack(t_vpr_setup& vpr_setup, const t_arch& arch);
+/** @brief Loads a previous packing */
+void vpr_load_packing(t_vpr_setup& vpr_setup, const t_arch& arch);
+
+/** @brief Perform, load or skip the placement stage */
+bool vpr_place_flow(t_vpr_setup& vpr_setup, const t_arch& arch);
+/** @brief Perform placement */
+void vpr_place(t_vpr_setup& vpr_setup, const t_arch& arch);
+/** Loads a previous placement */
+void vpr_load_placement(t_vpr_setup& vpr_setup, const t_arch& arch);
+
+/** @brief Perform, load or skip the routing stage */
+RouteStatus vpr_route_flow(t_vpr_setup& vpr_setup, const t_arch& arch);
+/** @brief Perform routing at a fixed channel width) */
 RouteStatus vpr_route_fixed_W(t_vpr_setup& vpr_setup,
```

Comment:
If we're going to put the comments on different lines, we should also put a blank line between these prototypes.

Reply:
I've added the blank lines.